### ISSUE TITLE
possible duplicate code in cli.py - run_create()

### ISSUE THREAD

**Issue by mengfp** [miurahr/py7zr#442]
**Describe the bug**

```python
    def run_create(self, args):
        sztarget = args.arcfile  # type: str
        filenames = args.filenames  # type: List[str]
        volume_size = args.volume[0] if getattr(args, "volume", None) is not None else None
        if volume_size is not None and not self._check_volumesize_valid(volume_size):
            sys.stderr.write("Error: Specified volume size is invalid.\n")
            self.show_help(args)
            exit(1)
        if not sztarget.endswith(".7z"):
            sztarget += ".7z"
        target = pathlib.Path(sztarget)
        if target.exists():
            sys.stderr.write("Archive file exists!\n")
            self.show_help(args)
            exit(1)
        if not args.password:
            password = None  # type: Optional[str]
        else:
            try:
                password = getpass.getpass()
            except getpass.GetPassWarning:
                sys.stderr.write("Warning: your password may be shown.\n")
                return 1

        ''' Duplicate ???
        with py7zr.SevenZipFile(target, "w", password=password) as szf:
            for path in filenames:
                src = pathlib.Path(path)
                if src.is_dir():
                    szf.writeall(src)
                else:
                    szf.write(src)
        '''

        if volume_size is None:
            with py7zr.SevenZipFile(target, "w") as szf:
                for path in filenames:
                    src = pathlib.Path(path)
                    if src.is_dir():
                        szf.writeall(src)
                    else:
                        szf.write(src)
            return 0
        else:
            size = self._volumesize_unitconv(volume_size)
            with multivolumefile.MultiVolume(target, mode="wb", volume=size, ext_digits=4) as mvf:
                with py7zr.SevenZipFile(mvf, "w") as szf:
                    for path in filenames:
                        src = pathlib.Path(path)
                        if src.is_dir():
                            szf.writeall(src)
                        else:
                            szf.write(src)
            return 0
```

**Related issue**

**To Reproduce**

**Expected behavior**

**Environment (please complete the following information):**
 - OS:  Windows 10
 - Python:  3.10
 - py7zr version:  0.18.5

**Test data(please attach in the report):**

**Additional context**

**Comment by miurahr** [comment]
Thank you for your findings. It seems to be a git merge error.

I've pushed a fix as #443 
Is #443 as same as your idea?

**Comment by miurahr** [comment]
The bug reported was created  when merge multivolume code with encryption support at https://github.com/miurahr/py7zr/commit/db7dee08295b66dc09f9b43ed229bcbd9dd5e2f9
